src/java.desktop/windows/classes/sun/awt/Win32GraphicsEnvironment.java

```diff
@@ -86,14 +88,10 @@ public final class Win32GraphicsEnvironment extends SunGraphicsEnvironment {
      * includes everything from the native GraphicsDevice elements to
      * the DirectX rendering layer.
      */
-    private static native void initDisplay();
+    private static native boolean initDisplay();
 
-    private static boolean displayInitialized;      // = false;
```

Comment:
So it seems like this is currently un-needed since there's only the one static initializer that calls this, so removing it should be fine.

Reply:
Exactly. There is just this one private usage.